src/client/components/ContactForm/ContactForm.component.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './ContactForm.styles.css';
+
+export default function ContactForm({user}) {
```

Comment:
I would destructure the props you need directly like this:

```suggestion
export default function ContactForm({fullName, email}) {
```

Reply:
Made changes according to the review comments